libs/ui/src/modals/screens/token/components/token-container/token-container.tsx

```diff
@@ -79,8 +76,7 @@ export const TokenContainer: FC<Props> = ({
               field={field}
               label="Token ID"
               placeholder="0"
-              prompt="What is Token ID?"
-              handlePrompt={handlePromptNavigate}
+              prompt="Enter the token ID description"
```

Comment:
Description looks weird or not?

Reply:
yes
but it's a text from a task..
I think we could discuss it on daily